package.json

```diff
@@ -68,6 +68,7 @@
         "simple-progress-webpack-plugin": "^1.1.0",
         "string-replace-loader": "^2.1.1",
         "webpack": "^3.11.0",
+        "webpack-assets-manifest": "1",
```

Comment:
Has semver gone a bit cray-cray?

Reply:
ha that's true! https://www.npmjs.com/package/webpack-assets-manifest#installation